client/src/partials/users/userPermission.modal.html

```diff
@@ -0,0 +1,41 @@
+<form name="PermissionForm" bh-submit="UserPermissionModalCtrl.submit()">
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="title">
+        {{ "USERS.PERMISSIONS" | translate }}
+        <label class="label label-warning">{{ "USERS.PERMISSIONS" | translate }}</label>
+        <label class="label label-warning">{{ UserPermissionModalCtrl.user.display_name }} ({{ UserPermissionModalCtrl.user.username }})</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+    <ul class="list-group" style="max-height: 600px; overflow:auto;">
+      <li class="list-group-item">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" ng-model="UserPermissionModalCtrl.super" ng-change="UserPermissionModalCtrl.toggleSuperUserPermissions(UserPermissionModalCtrl.super)">
+            {{ "USERS.SUPER" | translate }}
+          </label>
+        </div>
+      </li>
+      <li ng-repeat="unit in UserPermissionModalCtrl.units" class="list-group-item">
+        <div class="checkbox" ng-style="UserPermissionModalCtrl.checkboxOffset(unit.depth)">
```

Comment:
The spacing is really big for this modal, due to the combination of `list-group-item` and `checkbox` classes.  I propose that you had `style="margin:0"` to the `<div class="checkbox" ...>` DOM node.

In the image below, I've changed `Caisses et Banques` to using `margin:0` for comparison.  Let me know what you think.
![permissionsmargins](https://cloud.githubusercontent.com/assets/896472/18837036/3ddc1434-83fa-11e6-91bf-1ca9d8ab90af.png)
_Fig 1: Spacing Difference in `list-group-item`s_

If you think it is too small, you can pick a size smaller than 10px (which is what it is now) that you think looks best.